src/errors.d

```diff
@@ -18,6 +18,8 @@ import ddmd.globals;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 
+//debug = printGaggedErrors;
```

Comment:
I guess I don't see the point. One can comment/uncomment this line, or the comment/uncomment the fprintf. At least with the latter you aren't searching through the code wondering what `printGaggedErrors` does.